plugin/pkg/admission/limitranger/admission_test.go

```diff
@@ -634,3 +642,64 @@ func TestLimitRangerCacheAndLRUExpiredMisses(t *testing.T) {
 		t.Errorf("Should have ignored calls to any subresource of pod %v", err)
 	}
 }
+
+func validPersistentVolumeClaim(name string, resources api.ResourceRequirements) api.PersistentVolumeClaim {
+	pvc := api.PersistentVolumeClaim{
+		ObjectMeta: api.ObjectMeta{Name: name, Namespace: "test"},
+		Spec: api.PersistentVolumeClaimSpec{
+			Resources: resources,
+		},
+	}
+	return pvc
+}
+
+func TestPersistentVolumeClaimLimitFunc(t *testing.T) {
+	type testCase struct {
+		pvc        api.PersistentVolumeClaim
+		limitRange api.LimitRange
+	}
+
+	successCases := []testCase{
+		{
+			pvc:        validPersistentVolumeClaim("pvc-is-min-storage-request", getResourceRequirements(getStorageResourceList("1Gi"), getStorageResourceList(""))),
+			limitRange: createLimitRange(api.LimitTypePersistentVolumeClaim, getStorageResourceList("1Gi"), api.ResourceList{}, api.ResourceList{}, api.ResourceList{}, api.ResourceList{}),
```

Comment:
What is the established pattern for unsupported fields?  Silently ignored (docs discuss them) or fail?

I'd like to go with whatever standard is in place. If no standard, I am happy with silent ignoring. I suppose one could argue failure is explicit while silently ignoring is implicit and therefore not obvious.